tox.ini

```diff
@@ -1,10 +1,15 @@
 [tox]
-envlist =
+# Don't use "min_version", to ensure Tox 3 respects this
+minversion = 4
+env_list =
     lint,
     type
 
+[testenv]
+base_python = python3.10
+use_develop = true
```

Comment:
FYI, it's generally safe to run tests against an editable install for an application like this.

If we were building a library where we needed to guarantee that the sdist is correct, we'd want to be installing the package into an isolated test environment, to ensure that `import imagedephi` worked properly.